java/google/registry/rde/imports/RdeImportUtils.java

```diff
@@ -66,10 +83,11 @@ public RdeImportUtils(
     this.escrowBucketName = escrowBucketName;
   }
 
```

Comment:
OK, we'll see what happens when we do the import.  They might end up becoming static anyway if our static code analysis tool ends up complaining.